src/Azure/Orleans.Streaming.EventHubs/Hosting/DeveloperExtensions.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+
+namespace Orleans.Hosting.Developer
```

Comment:
I feel these should still be in `Orleans.Hosting` - the `.Developer` ns will make them very difficult to discover

Reply:
Understood.  These are not surfaces intended for production use.  They exist for developers and test.  Being less discoverable was the intention.  So I suspect the question we need resolve here is whether they should be less discoverable?

Reply:
I'm not sure. It's a hard question. A part of me wonders: should we be checking some config flag? Eg, something like `IHostEnvironment.IsDevelopment`, or maybe something more Orleans-specific?

That's not an immediate solution... but in the short term maybe we could fix this with regular naming. My feeling is that we should make the functionality we offer discoverable but I don't know the use-case for generators in order to decide what value they have: if they are just for dev/test then I'm happy for no one to know about them (and even happier for us to delete them).